test/docs/testplans/vnet_to_vnet.md

```diff
@@ -0,0 +1,54 @@
+[TOC]
+
+# **VNET to VNET**
+
+## **Overview**
+
+This test plan corresponds to the phase where standardized, automated test cases will be scripted to verify Data plane Testing Stage 3 as outlined in [dash test maturity stages](https://github.com/PLVision/DASH/blob/test-framework-extension/test/docs/dash-test-maturity-stages.md#data-plane-testing-stage-2-standardized-automated-test-cases). In these tests we want to levarage the snappi API to test some of the functional aspects of DUT. By verifying and validating the scaling aspects of the switches (hardware/BMv2) we are able to emperically baseline the scaling values. Additonally, we also want to check some of the relevant counters to features like IPv4, IPv6, TCP, UDP, common traffic stats etc.
+
+### **Scope**
+
+These test cases are targeted for testing the feature functionality of DASH implementation. The purpose of these test cases are to verify inbound/outbound/bidirectional traffic streams originating and terminating at the traffic generator. We plan to use the OTG (Open traffic generator model) by leveraging the snappi APIs. 
```

Comment:
Make Open Traffic Generator Model a link to the Github site e.g. https://github.com/open-traffic-generator